src/quicktype-core/language/Python.ts

```diff
@@ -257,7 +258,11 @@ export class PythonRenderer extends ConvenienceRenderer {
     }
 
     protected namerForObjectProperty(): Namer {
-        return funPrefixNamer("property", s => snakeNameStyle(this.pyOptions.features.version, s, false));
+        if (this.pyOptions.nicePropertyNames) {
+            return funPrefixNamer("property", s => snakeNameStyle(this.pyOptions.features.version, s, false));
+        } else {
+            return funPrefixNamer("properties", s => s);
+        }
```

Comment:
This appears to change the default output of quicktype, which will break existing users' code. We prefer not to do this.